c2corg_ui/templates/outing/edit.html

```diff
@@ -94,11 +94,12 @@ <h5 translate ng-show="outing.associations.routes.length == 0">choose at least o
             </div>
 
             ## PARTIAL TRIP
-            <div class="form-group data half" ng-class="{'has-success': outing.partial_trip }">
+            <div class="form-group data half" ng-class="{'has-success': outing.partial_trip == false || outing.partial_trip == true }">
```

Comment:
Oops, I forgot to have a look at this commit :P

What about ``outing.partial_trip !== undefined`` instead of ``outing.partial_trip == false || outing.partial_trip == true``?